index.html

```diff
@@ -129,5 +129,72 @@ <h1 class="header-logo-wrapper">
         </a>
       </div>
     </section>
+    <section class="signup">
+      <div class="signup-container">
+        <p class="signup-section-head">Let's Try!</p>
+        <p class="signup-section-body">
+          Grand
+          Playersでは、ご利用登録いただけた方に対し、3レッスン分の無料枠をプレセントいたします!
+          <br
+            class="PC-br"
+          />今すぐレッスンを初めて、”Grand”なプレイヤーを目指しましょう!
```

Comment:
```suggestion
　　今すぐレッスンを初めて、”Grand”なプレイヤーを目指しましょう!
```
こちらfigma側のミスなのでなくても大丈夫です 🙇 

クラス名に関しては、ケバブケースで `pc-br` とするのが適切かと
もしくは、 `only-pc` みたいな汎用的なクラス名としてしまうのもありですね